astropy/coordinates/builtin_frames/enu.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+import numpy as np
+
+import astropy.units as u
+import astropy.time as at
+
+from astropy.coordinates.baseframe import (BaseCoordinateFrame, RepresentationMapping,                               frame_transform_graph)
+
+from astropy.coordinates.attributes import (TimeAttribute, EarthLocationAttribute)
+
+from astropy.coordinates.transformations import FunctionTransform
+from astropy.coordinates.representation import (SphericalRepresentation,
+                              UnitSphericalRepresentation,CartesianRepresentation)
+from . import ITRS
+
+class ENU(BaseCoordinateFrame):
+    """
+    A coordinate or frame in the East-North-Up (ENU) system.
```

Comment:
I think a bit more specifics about this system are called for.  It's centered on the *surface* of the earth, right? (at `location`)  and the `east`/`north`/`up` are based on the `obstime` orientation?  So is it different from `AltAz` in any way other than the names?  If so, it should be tied to `AltAz` rather than `ITRS`, right?

Reply:
(or if I'm misunderstanding and it's different from `AltAz`, that difference should be stated clearly here in the docstring)

Reply:
So the `ENU` system and `AltAz` have similar coordinates except `AltAz` is left-handed. Azimuth increases from North to East. AltAz is the same as North-East-Down. Otherwise the origin is fixed to `location` like `AltAz`. I included `ENU` because I had found it useful before, however I'm not married to the idea of including it in `astropy`. However, I think the `UVW` is certainly useful for many people. I chose to tie the graph transform to `ITRS` for simplicity in terms of rotations but now that I think about it, simply flipping Up to Down and rearranging `ENU` to `NED` is simpler.

Reply:
My main concern here is that there's some subtle differences between `AltAz` and the `ENU` implemented here - the most obvious is the possibility of atmospheric refraction, but there are more subtle things (like the different code pathways having different machine precision since `AltAz` uses erfa).  So Might it make more sense for this to be transformed from `AltAz`? That comes with some baggage, though, of requiring an atmospheric refraction.

So I'm tempted to instead say either drop `ENU` entirely or have it be tied to `UVW`. (I *think* the latter is what you were suggesting in your last sentence here?)

Reply:
So I've decided I should just drop `ENU` as it's not strictly an astro frame in common use. In anycase, it's the same as UVW with `phase = zenith`.